desc/vmec.py

```diff
@@ -258,6 +255,62 @@ def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
         file.createDimension("dim_00100", 100)
         file.createDimension("dim_00200", 200)
 
+        # compute data
+        timer.start("compute")
+        if verbose > 0:
+            print("Computing data")
+
+        # desc throws NaN for r == 0, so we use something small to approximate it
```

Comment:
I thought that we have axis limits implemented for most things?

Reply:
I think this was left over from before we implemented the axis limits. Updated now, thanks.